chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -66,6 +66,37 @@
   supports restart: true
   action [:enable, :start]
   subscribes :restart, "template[/etc/ceph/ceph.conf]"
+  notifies :run, "execute[trigger_radosgw_metering_setup]", :delayed
+end
+
+execute "trigger_radosgw_metering_setup" do
+  command "true"
+  notifies :run, "execute[create_radosgw_admin_user]", :delayed
+  notifies :run, "execute[add_radosgw_buckets_caps]", :delayed
+  notifies :run, "execute[add_radosgw_users_caps]", :delayed
+end
+
+# Create admin user and add caps to access buckets and users
+uid = "rgw_admin"
+
+execute "create_radosgw_admin_user" do
```

Comment:
On this block chef-client is locking during deployment, maybe radosgw is not ready